### PR TITLE
Add json and code  of edx.video.stopped event

### PR DESCRIPTION
**Pull request by tehreem-sadat**
**Trello Link:** _https://trello.com/c/prEbw763/7-video-interaction-event-stopvideo-edxvideostopped_

**Description:** _When the video player reaches the end of the video file and play automatically stops, the player emits a stop_video event._

**Checks before merge:**

- [x] Reviewed
- [x] Unit Test
- [x] Code Reviewed
- [x] Commits squashed

### CODE REVIEW

**Comment by tehreem-sadat** [comment]
@asamolion Json has already been approved by aroosha mam. Kindly do a code review.

File: openedx/features/caliper_tracking/tests/current/edx.video.stopped.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "accept_language": "en-US,en;q=0.5",
```
Comment (asamolion) [inline openedx/features/caliper_tracking/tests/current/edx.video.stopped.json]: Please rename both JSON files to `stop_video.json`

Comment (tehreem-sadat) [reply openedx/features/caliper_tracking/tests/current/edx.video.stopped.json]: Done (y). Kindly review it again.

File: openedx/features/caliper_tracking/tests/expected/stop_vieo.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
```
[the remark this answers is not in the document]
Comment (tehreem-sadat) [reply openedx/features/caliper_tracking/tests/expected/stop_vieo.json]: done

**Pull request merged**